common/workunit/workunit.cpp

```diff
@@ -6847,25 +6848,39 @@ IStringVal& CLocalWUQuery::getQueryText(IStringVal &str) const
 
 IStringVal& CLocalWUQuery::getQueryShortText(IStringVal &str) const
 {
-    const char * text = p->queryProp("Text");
```

Comment:
These changes are trying to make the Dali version more efficient - it won't actually make a lot of difference in Cassandra as yet (unless I make the query text even more lazy loaded - which I may)